01week/datatypes.js

```diff
@@ -0,0 +1,76 @@
+ 'use strict';
+
+<!-- #1 - Display the Date -->
+
+function ReturnDate() {
+const date = new Date;
```

Comment:
the const in this instance is redundant, this can be simplified to one line `return new Date;`